dagobah/backend/base.py

```diff
@@ -41,12 +40,7 @@ def verify_required_packages(self):
                                                                                                         spec['version']))
                 continue
 
-            # everyone has to be special
-            if spec['pypi_name'] == 'pysqlite':
-                module = __import__('pysqlite2._sqlite')
-                installed_version = getattr(module._sqlite, spec['version_key'])
-            else:
-                installed_version = getattr(module, spec['version_key'])
+            installed_version = getattr(module, spec['version_key'])
```

Comment:
I did this part somewhat blindly, but I think this is correct now that SQLite is out of the picture?